.changeset/twenty-shoes-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/database": patch
+---
+
+Fix websocketonly field to be set to true when using websocket db url.
```

Comment:
Could you explain what the websocket url is in parens (ws or wss instead of https)? Then someone can figure out how to use it by reading the release notes.